AnimationGroup.lua

```diff
@@ -24,7 +24,7 @@
 
 if not LibStub then return end
 
-local MAJOR_VERSION, MINOR_VERSION = 'AnimationGroup-1.0', '$Format:%ct-%h$'
+local MAJOR_VERSION, MINOR_VERSION = 'AnimationGroup-1.0', '1520787716-0b498e1'
```

Comment:
```suggestion
local MAJOR_VERSION, MINOR_VERSION = 'AnimationGroup-1.0', '$Format:%ct-%h$'
```

This should only change if a distribution is released.

As for as I remember, the `release` script does the replacing. It's Bash, so you would probably have to call it from WSL if you are on Windows.

Reply:
Im changing it in game - to load newest version =)

Reply:
Sure - but don't commit these changes :)